src/schema/v2/fields/markdown.ts

```diff
@@ -2,35 +2,44 @@ import Format from "schema/v2/input_fields/format"
 import { GraphQLString, GraphQLFieldConfig } from "graphql"
 import { isExisty } from "lib/helpers"
 import marked from "marked"
+import { ResolverContext } from "types/graphql"
 
-export function formatMarkdownValue(value, format) {
+type Format = "html" | "markdown"
+type Value = string | null | undefined
+
+export const formatMarkdownValue = (value: string, format: Format): string => {
   if (format === "html" || format === "markdown") {
     const renderer = new marked.Renderer()
+
     marked.setOptions({
       renderer,
       gfm: true,
-      tables: true,
       breaks: true,
       pedantic: false,
       sanitize: false,
       smartypants: false,
+      tables: true,
```

Comment:
This option gets dropped in the 1.0.0 release of marked, if we ever want to upgrade.